### ISSUE TITLE
Missing schema in display properties

### ISSUE THREAD

**Issue by DRK3** [decentralized-identity/credential-manifest#65]
Hi!

In [Section 4.2.1](https://identity.foundation/credential-manifest/#output-descriptor-object) it says that

> The Output Descriptor Object MAY contain a display property, and its value MUST be an object composed as defined in the [DIF Data Display](https://identity.foundation/credential-manifest/wallet-rendering/#data-display) specification.

According to the linked Wallet Rendering spec, it looks like when [you're using a path, an object must contain a schema property](https://identity.foundation/credential-manifest/wallet-rendering/#using-path). I noticed the examples here in Credential Manifest don't have it. I'm wondering if the omission is intentional or not.

(If it's not intentional, I could make a PR to add it in to the examples)

**Comment by JaceHensley** [comment]
Good catch! I think the examples are just out of date, the `schema` property is required when using `path`

**Comment by JaceHensley** [comment]
We are actually moving the Wallet Rendering spec to it's own repo https://github.com/decentralized-identity/wallet-rendering
